app/helpers/record_helper.rb

```diff
@@ -21,13 +21,39 @@ def paginated_more_like_this(response, similar)
     }
   end
 
+  def record_hierarchy(hierarchy)
```

Comment:
Assignment Branch Condition size for record_hierarchy is too high. [20.1/15]<br>Method has too many lines. [17/10]